### PR TITLE
Update README.mdown

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: generate_dependent_styles/README.mdown [hunk 1]
```diff
@@ -87,12 +88,16 @@ An example of `_rename.tab` is shown below:
...
 
 Running the script with the `--dir` (or `-d`) option allows you to limit style generation to a single `data` subdirectory (e.g., `ruby generate_styles.rb -d asm`).
 
-Running the script with the `--replace` (or `-r`) option will delete any dependent styles from the `styles` directory that have a matching XML comment ("Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/**subdirectory**"), and copy over the newly generated styles. This only works if the `styles` and `utilities` repositories are checked out to the same root directory.
+Running the script with the `--replace` (or `-r`) option will delete any dependent styles from the `styles` directory that have a matching XML comment ("Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/**subdirectory**"), and copy over the newly generated styles. This only works if the `styles` and `utilities` repositories are checked out to the same root directory.
```
Comment (rmzelle) [inline generate_dependent_styles/README.mdown]: Thanks!

I still have to move over the metadata to its own repository, and until then I'll keep the "Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/**subdirectory**" XML comment as is. Ultimately I would like to use something shorter like "".

Comment (zuphilip) [inline generate_dependent_styles/README.mdown]: Okay, I revert that line back.

**Comment by rmzelle** [comment]
Oh, and in case you use the "--replace" option of the script: I introduced the "--force" option. Now, by default, styles aren't replaced if only their timestamp changed. Using "--force" forces the replacement in all cases.

File: generate_dependent_styles/README.mdown [hunk 2]
```diff
@@ -87,8 +88,12 @@ An example of `_rename.tab` is shown below:
 
 ## Running generate_styles.rb
 
+### Generating styles of all publishers
+
 The Ruby `generate_styles.rb` script can be run without arguments. In this case, the script will generate styles for each directory within `data`, and place the generated styles in matching subdirectories at `generate_dependent_styles/generated_styles`.
```
Comment (rmzelle) [inline generate_dependent_styles/README.mdown]: The output folder changed from "generate_dependent_styles/generated_styles" to "generate_dependent_styles/dependent" as well.

Comment (rmzelle) [inline generate_dependent_styles/README.mdown]: (and I can make those changes as well in the README; I was just a bit lazy)

Comment (zuphilip) [inline generate_dependent_styles/README.mdown]: I have to read and understand the README anyway for the Springer Update 2016. Everything which I notice in the description I will suggest the improvements here. Yeah, these two sections are quite hard for me to understand and maybe I can rewrite them (and we should deal with all changes of the path as well). Feel free to jump in and help with rewriting it, if your time permits.

Comment (rmzelle) [inline generate_dependent_styles/README.mdown]: You can also run the script with "--help" to get some guidance.

**Comment by zuphilip** [comment]
Small Update. (Replacing und Forcing ares still missing some update.)

**Comment by rmzelle** [comment]
Thanks! I made a few more edits: https://github.com/citation-style-language/utilities/commit/5461fbd190aca29bb20e46b0d5193db083566f01

**Comment by zuphilip** [comment]
Thank you for additions! These edits are very much appreciated!

**Comment by rmzelle** [comment]
Thanks for pushing this! I noticed I never documented the "replace" options ("additions", "deletions", "modifications"), which I've found to be quite useful in preparing pull requests to the "styles" repo, especially for big updates like the Springer styles. Git sometimes gets confused when we both delete and add new styles (and then thinks we renamed the styles), and with these options you can just create three separate commits, one for style additions, one for deletions, and one for modifications of existing styles.

**Comment by zuphilip** [comment]
Oh, yeah! We should add this information for the application as well. I will prepare a PR by stealing your sentences here...;-)

**Pull request merged**